text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/transformation/TransformationType.java

```diff
@@ -182,4 +190,58 @@ private TransformationType(final Predicate<String> canParse, final Transformatio
       return names::contains;
     }
   }
+
+  /**
+   * Create a new color transformation type for parsing the supplied color map.
+   *
+   * @param aliases map of color names to their {@link TextColor} values
+   * @return a new color transformation
+   * @since 4.10.0
+   */
+  public static TransformationType<ColorTransformation> color(final Map<String, TextColor> aliases) {
```

Comment:
I think these  methods should be in color transformation and the ctor there private